src/implementations/twig/components/site-footer/ec-site-footer.story.js

```diff
@@ -0,0 +1,140 @@
+import { withNotes } from '@ecl/storybook-addon-notes';
+import withCode from '@ecl/storybook-addon-code';
+import { correctSvgPath } from '@ecl/story-utils';
+
+import dataCore from '@ecl/specs-component-site-footer/demo/data-core--ec';
+import dataStandardised from '@ecl/specs-component-site-footer/demo/data-standardised--ec';
+import dataHarmonised from '@ecl/specs-component-site-footer/demo/data-harmonised--ec';
+import logoEc from '@ecl/resources-ec-logo/negative/logo-ec--en.svg';
+import footer from './site-footer.html.twig';
+import notes from './README.md';
+
+const getArgs = () => {
+  const args = {
+    hide_contact: true,
```

Comment:
in the other component we name the switcher the other way ("show_contact" instead of "hide_contact")

Reply:
What component are you referring to? I didn't really touch the controld, I kept the same logic, but here all the controls are in hide mode? would you like me to put them all in show mode?

Reply:
Yes, we didn't update the control of headers/footer when doing the global cleanup a few weeks ago, as we knew that they would be modified soon.
You can check for instance the content item (https://github.com/ec-europa/europa-component-library/blob/v3-dev/src/implementations/twig/components/content-item/content-item.story.js) or the card (https://github.com/ec-europa/europa-component-library/blob/v3-dev/src/implementations/twig/components/card/card.story.js)